static-assets/components/cstudio-dialogs/crop-dialog.js

```diff
@@ -449,6 +455,8 @@ CStudioAuthoring.Dialogs.CropDialog = CStudioAuthoring.Dialogs.CropDialog || {
       if (newFileName) {
         const newFullPath = `${path}/${newFullName}`;
         const createNewNameFile = () => {
+          imageData.renameRelativeUrl = `${imageData.relativeUrl.replace(fileName, newFullName)}`;
```

Comment:
Restored this lines. imageData seems to be a reference, so by updating those values, when saving the control checks for those fields and uses them if exist (on the crop callback)